models/jwtblock.js

```diff
@@ -0,0 +1,10 @@
+import { model, Schema } from "mongoose";
+
+const jwtBlock = new Schema({
+  "blockdate": String,
```

Comment:
Doublequotes not required!

Reply:
@rycont Thank you for helpful information